f2/src/locales/en.json

```diff
@@ -21,20 +21,16 @@
   "confirmationPage.backButton": "Back to Your contact details",
   "confirmationPage.businessInfo.business": "Description",
   "confirmationPage.businessInfo.title": "Business",
-  "confirmationPage.businessInfo.title.edit": "Edit Affected Business Assets",
```

Comment:
🙁 All these `.edit` lines shouldn't be deleted. There was a bug in a previous version of master that caused this 😢 
Can you add these lines back in?